States/SelectVenueToCloseState.cs

```diff
@@ -57,7 +57,7 @@ public async Task Handle(MessageComponentInteractionContext c)
             _ = c.Session.ClearState(c);
 
             await _apiService.CloseVenueAsync(venue.Id);
-            await c.Interaction.RespondAsync(MessageRepository.VenueClosedMessage.PickRandom());
+            await c.Interaction.FollowupAsync(MessageRepository.VenueClosedMessage.PickRandom());
```

Comment:
You can't respond more than once. I found a bug where there was already a hidden response sent back to discord before this confirmation message was sent back - so it errored. Changed it to `FollowupAsync` which allows more responses after the first. (Same with the instances below)

Reply:
I think I got it, but you can explain me more in VC later if you can xD